internal/pkg/agent/application/paths/paths_darwin.go

```diff
@@ -17,6 +17,9 @@ const (
 	// ControlSocketPath is the control socket path used when installed.
 	ControlSocketPath = "unix:///var/run/elastic-agent.sock"
 
+	// ControlSocketNonRootPath is the control socket path used when installed as non-root.
+	ControlSocketNonRootPath = "unix:///var/run/elastic-agent/elastic-agent.sock"
```

Comment:
could you add a comment why we need to place it in subdir, i'm sure we'll forget

Reply:
Added.